tests/c/debug_simple_interp_loop2.c

```diff
@@ -42,7 +42,7 @@
 //       %{{fptr2}} = getelementptr %YkCtrlPointVars, ptr %0, i32 0, i32 0...
 //       store...
 //       ...
-//       ret ptr null
+//       br...
```

Comment:
why don't we match the label name?

Reply:
We might change the label at some point? There is a test that checks this while being agnostic about the chosen block name: [tests/c/simple.c](https://github.com/ykjit/yk/pull/736/commits/36a65c103cb4051b5d5dedcd23ef2f0f96440124#diff-30045133b33c3db0b2bd22080a981b5cd9d1c5d79f8b1b3cf50c2bc2cbdec05f)

Reply:
Just this one and then I think we'll be ready to go.

Reply:
I more meant: shouldn't we do it everywhere?

Reply:
I just did the minimal change to get the test working and am leaning towards one place is enough to check this. But it's not a hill I'm going to die on, if you think it's useful.

Reply:
Fair enough.